src/extrawidgets.jl

```diff
@@ -123,20 +123,8 @@ function player(cs::Observable, range::AbstractUnitRange{<:Integer}; style="with
     error("style $style not recognized")
 end
 
-Base.unsafe_convert(::Type{Ptr{Gtk.GLib.GObject}}, p::PlayerWithTextbox) =
-    Base.unsafe_convert(Ptr{Gtk.GLib.GObject}, frame(p))
-
-function Gtk.destroy(p::PlayerWithTextbox)
-    destroy(p.play_back)
-    destroy(p.step_back)
-    destroy(p.stop)
-    destroy(p.step_forward)
-    destroy(p.play_forward)
-    destroy(p.entry)
-    destroy(p.scale)
-    destroy(frame(p))
-end
```

Comment:
I take it these are handled automatically now?

Reply:
Yeah, there's no `destroy` for widgets now, just for windows. If you destroy windows typically all the widgets are destroyed too.